### PR TITLE
Function to read group details from identity/group and refactored oidc functions to utils

### PR DESCRIPTION
**Pull request by sarathmohan4**

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/OIDCGroup.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+import java.io.Serializable;
+import java.util.List;
+
+
+public class OIDCGroup implements Serializable {
+
+	private static final long serialVersionUID = -1720191110593009341L;
+
+	private String id;
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/OIDCGroup.java:28]: Will it be useful to add group name also?

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/controller/OIDCUtil.java [hunk 2]
```diff
@@ -86,4 +116,170 @@ public static OIDCEntityResponse getEntityLookUpResponse(String authMountRespons
...
 
+	/**
+	 * To get identity group details.
+	 * @param groupName
+	 * @param token
+	 * @return
+	 */
+	public static OIDCGroup getIdentityGroupDetails(String groupName, String token) {
+		Response response = reqProcessor.process("/identity/group/name", "{\"group\":\""+groupName+"\"}", token);
+		OIDCGroup oidcGroup = new OIDCGroup();
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/controller/OIDCUtil.java:127]: Either you can initialize with null (not required) or move it inside if condition

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/OIDCGroup.java (hunk 1 above)
Comment (sarathmohan4) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/OIDCGroup.java:28]: Places we are using this model will have group name from the request itself, we can keep it, but i think that will not be used

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/controller/OIDCUtil.java (hunk 2 above)
Comment (sarathmohan4) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/controller/OIDCUtil.java:127]: Changed and updated in new PR

**Pull request merged**